presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -7230,6 +7276,22 @@ public void testMergeEmptyNonEmptyApproxSet()
         assertEquals(actual.getMaterializedRows(), expected.getMaterializedRows());
     }
 
+    @Test
+    public void testMergeEmptyNonEmptyApproxSetWithSameMaxError()
+    {
+        MaterializedResult actual = computeActual("SELECT cardinality(merge(c)) FROM (SELECT create_hll(custkey, 0.1) c FROM orders UNION ALL SELECT empty_approx_set(0.1))");
+        MaterializedResult expected = resultBuilder(getSession(), BIGINT)
+                .row(1046L)
+                .build();
+        assertEquals(actual.getMaterializedRows(), expected.getMaterializedRows());
+    }
+
+    @Test(expectedExceptions = IllegalArgumentException.class, expectedExceptionsMessageRegExp = "Cannot merge HLLs with different number of buckets.*")
+    public void testMergeEmptyNonEmptyApproxSetWithDifferentMaxError()
+    {
+        computeActual("SELECT cardinality(merge(c)) FROM (SELECT create_hll(custkey, 0.1) c FROM orders UNION ALL SELECT empty_approx_set(0.2))");
+    }
+
```

Comment:
Can you add a test for error outside the bounds, like 1?  Also if it throws `IllegalArgumentException`, can you repackage it so it is a Presto exception?